src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpointESSecurityIT.java

```diff
@@ -451,8 +451,7 @@ void shouldOnlyReturnCasesSolicitorHasBeenGrantedAccessTo() throws Exception {
         CaseSearchResult caseSearchResult = executeRequest(searchRequest, CASE_TYPE_C, AUTOTEST1_SOLICITOR);
 
         assertAll(
-            () -> assertThat(caseSearchResult.getTotal(), is(1L)),
-            () -> assertThat(caseSearchResult.getCases().get(0).getReference(), is(1589460125872336L))
```

Comment:
Is it possible to keep assertions verifying the exact cases please?

Reply:
Assertions has been changed.